### ISSUE TITLE
Screen reader is not announcing additions to an aria-live region.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#16697]
### Steps to reproduce:
1. Go to https://jsfiddle.net/j4e6ugdw/1/
2. Activate NVDA
3. Click on the button to add announcements.
4. Observe the issue.

### Actual behavior:

NVDA does not announce the content that appears in aria-live region

### Expected behavior:

NVDA should announce the additions to the aria-live region

### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/88aeb4f3-cae9-40ff-885e-ddac85e9a261

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2024.1.0.31547

#### Windows version:
Windows 11 23H2 22631.3672

#### Name and version of other software in use when reproducing the issue:
NA 

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NA

#### If NVDA add-ons are disabled, is your problem still occurring?
NA

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
NA

**Comment by seanbudd** [comment]
Closing as duplicate of #15857
